core/scheduler/service/executor_input_compiler.go

```diff
@@ -62,6 +63,33 @@ type InputCompiler struct {
 	logger log.Logger
 }
 
+func sanitiseLabel(key string) string {
+	// implements validation from https://cloud.google.com/bigquery/docs/labels-intro#requirements
+
+	// max length 63 chars
+	// if length greater than 63 then truncate to 61 and add __ at end to denote truncation
+	if maxLabelLength := 63; len(key) > maxLabelLength {
+		key = "__" + key[len(key)-61:]
```

Comment:
the description says that it will be `__` at the end, assuming it will be something like `some-label-not-don--`. But, the implementation says the `__` is put at the beginning. which one is correct?

Reply:
Also, please remove the comment if it's already clear from the implementation or comment of the requirement reference

Reply:
okay